string object_category_request

---
ObjectList object_list

